scripts/build_utils.sh

```diff
@@ -1373,6 +1377,11 @@ setup_rpm_build_deps() {
         # before EPEL8 and PowerTools provide all dependencies, we use sepia for the dependencies
         $SUDO dnf config-manager --add-repo http://apt-mirror.front.sepia.ceph.com/lab-extras/8/
         $SUDO dnf config-manager --setopt=apt-mirror.front.sepia.ceph.com_lab-extras_8_.gpgcheck=0 --save
+
+	$SUDO dnf install -y epel-release || sudo dnf install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1).noarch.rpm
+	$SUDO dnf install -y https://apache.jfrog.io/artifactory/arrow/centos/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm
```

Comment:
again, i think we just need to
```sh
$SUDO dnf config-manager --add-repo="https://apache.jfrog.io/artifactory/arrow/centos/$RELEASE}/$(arch)"
```

Reply:
ok, thanks.
will try that (containerized environment).